dev/io.openliberty.org.jboss.resteasy.common/bnd.bnd

```diff
@@ -114,21 +112,27 @@ Export-Package: \
 # Using version=! in order to not have a version attached to the import for packages that were removed
 # from Java after Java 8.  Doing this keeps the import like before Java 11 support. It will get the
 # packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
+# We have to manually exclude the jakarta.ws.rs and jakarta.servlet packages and rely on the tranformer to transform the javax versions.
+# This is to avoid including the jakarta verisons multiple times, resulting in a compilation error.
 Import-Package: \
-  javax.activation;version=!, \
-  javax.annotation;version=!, \
-  javax.annotation.security;version=!, \
-  javax.xml.bind;version=!, \
-  javax.xml.bind.annotation;version=!, \
-  javax.xml.bind.annotation.adapters;version=!, \
-  javax.xml.bind.attachment;version=!, \
-  javax.xml.bind.helpers;version=!, \
-  javax.xml.bind.util;version=!, \
-  !javax.json.bind, \
-  !javax.json.bind.spi, \
-  !javax.mail, \
-  !javax.mail.internet, \
-  !javax.mail.util, \
+  !jakarta.ws.rs*, \
+  !jakarta.servlet*, \
+  javax.ws.rs*, \
+  javax.servlet*, \
+  jakarta.activation;version=!, \
+  jakarta.annotation;version=!, \
+  jakarta.annotation.security;version=!, \
+  jakarta.xml.bind;version=!, \
+  jakarta.xml.bind.annotation;version=!, \
+  jakarta.xml.bind.annotation.adapters;version=!, \
+  jakarta.xml.bind.attachment;version=!, \
+  jakarta.xml.bind.helpers;version=!, \
+  jakarta.xml.bind.util;version=!, \
```

Comment:
I think all of these can be removed.  The statement about activation, annotation and xml.bind.* isn't accurate for jakarta.  Also the ws.rs and servlet will be wrong without the transform happening any longer.  Also the comment should be removed.